output.py

```diff
@@ -0,0 +1,44 @@
+import os
+import jinja2
+
+from const import file_name_str, report_file_path, report_css_path, audio_img_url_str, video_img_url_str
+
+template_dir = os.path.join(os.path.dirname(__file__), 'templates')
+jinja_env = jinja2.Environment(loader=jinja2.FileSystemLoader(template_dir), autoescape=True)
+
+
+# output to file
+def write_to_report_folder(file_name, folder_path, stream_info, bitrate_div, qp_div, c_plot):
+    """
+    generate report folder
+    :param file_name: name of input video
+    :param folder_path: output report folder path
+    :param stream_info: stream info dict
+    :param bitrate_div: bitrate graph div
+    :param qp_div: qp graph div
+    """
+    if not os.path.exists(folder_path + "/report"):
+        os.mkdir(folder_path + "/report")
+    write_to_html(file_name, folder_path, stream_info, bitrate_div, qp_div, c_plot)
+
+
+def write_to_html(file_name, folder_path, stream_info, bitrate_div, qp_div, c_plot):
+    """
+    write the information to HTML report
+    :param file_name: name of input video
+    :param folder_path: output report folder path
+    :param stream_info: video stream info dict
+    :param bitrate_div: bitrate graph div
+    :param qp_div: qp graph div
+    """
+    report = jinja_env.get_template('report.html')
+    audio_img_url = os.getcwd() + '/audio-icon.jpg'
```

Comment:
You should write these to the report folder. Feel free to do in follow up cl.

Reply:
Also note that user may be running this from a different folder, so os.getcwd() isnt what you want anyway.

Reply:
Yes. I put all the support files (img, data, js and css) to the report folder so html report can import from there.